packages/flutter_svg_icon/pubspec.yaml

```
@@ -14,4 +16,15 @@
+    sdk: flutter
+
+topics:
+  - svg
+  - icon
+  - svg-icon
+  - flutter-svg
+  - flutter-icon
+
+screenshots:
+  - description: 'Flutter SVG Icon Showcase'
+    path: assets/showcase.png
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)